internal/api/genapi/templates.go

```diff
@@ -266,7 +266,49 @@ func (c *Client) List(ctx context.Context, {{ .CollectionFunctionArg }} string,
 		return nil, apiErr
 	}
 	target.response = resp
-	return target, nil
+	if opts.withRefreshToken != "" || target.ResponseType == "complete" {
+		return target, nil
+	}
+	// if refresh token is not set explicitly and there are more results,
+	// automatically fetch the rest of the results.
+	for {
+		req, err := c.client.NewRequest(ctx, "GET", "targets", nil, apiOpts...)
```

Comment:
Should this use a template variable?

```suggestion
		req, err := c.client.NewRequest(ctx, "GET", "{{ .CollectionPath }}", nil, apiOpts...)
```

Reply:
Good spot 😁